neuro_san/service/http/handlers/base_request_handler.py

```diff
@@ -162,14 +164,19 @@ def data_received(self, chunk):
         """
         return
 
-    def prepare(self):
+    # Tornado can handle both syns and async versions of "prepare" method
+    # pylint: disable=invalid-overridden-method
+    async def prepare(self):
         if not self.application.is_serving():
```

Comment:
Tornado can handle both sync and async methods thanks to internal magic machinery,
but async is "native" to its event loop.